### DISCUSSION TITLE
Filtering Prometheus alerts from Prometheus data source by Org/User

### DISCUSSION THREAD

**Discussion by wkneewalden** [grafana/grafana#52101] [Alerting]
Currently all Prometheus alerts associated with a Prometheus data source appear in the Alerting Rule tab. Is it possible to have a filter like regex to control which ones are viewable in a given organization, assuming multiple organizations share the same data source? Can permissions control which users observe them? The idea is to limit viewing access for users to only alerts relevant to their organization from that data source.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
